### PR TITLE
Add missing LineEdit constants in editor theme

### LINKED ISSUE #76119: Project theme LineEdit override breaks some LineEdits in Editor
### Godot version

4.0.1, 4.0.2, probably 4.0 (Mono version, not sure about regular)

### System information

Windows 10

### Issue description

Setting LineEdit override in default theme in project settings (not editor settings!) affects editor settings. It breaks some of the LineEdits in editor (see screenshot 1), making their `minimum_character_width` equal to 0 (default value in import without data). 
diabc2y5g6
Affected components are ColorPicker and Sprite2D region editor (in grid mode). It might also affect Tilemap editor too. Setting `minimum_character_width` to some value (3, for example) "fixes" editor too, and it looks like other settings are not affecting editor theme (at least style overrides, see screenshot 2).
Godot_v4 0 2-stable_mono_win64_Wt5CaTFSSo

### Steps to reproduce

1. Create new project.
2. Add new theme.
3. Set it as default theme in Project settings (not Editor settings!).
4. Restart Godot (as requested by UI).
5. Add LineEdit override. 
6. Change scene in Editor or restart it.
7. Try to open ColorPicker color selection menu or Region editor of Sprite2D.

### Minimal reproduction project

[ProjectThemeLineEditBug.zip](https://github.com/godotengine/godot/files/11241546/ProjectThemeLineEditBug.zip)

### PR DESCRIPTION
**Pull request by KoBeWi**
Fixes #76119

### CODE REVIEW

File: editor/editor_themes.cpp [hunk 1]
```diff
@@ -1464,6 +1467,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 
 	theme->set_constant("line_spacing", "TextEdit", 4 * EDSCALE);
 	theme->set_constant("outline_size", "TextEdit", 0);
+	theme->set_constant("caret_width", "LineEdit", 1);
```
Comment (YuriSizov) [inline editor/editor_themes.cpp:1470]:
```suggestion
	theme->set_constant("caret_width", "TextEdit", 1);
```

Comment (KoBeWi) [reply editor/editor_themes.cpp:1470]: 🙃

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.3.

**Pull request merged**